.travis.yml

```diff
@@ -15,14 +15,30 @@ before_install:
 # Don't build on tags because we'll already have built the commit.
 jobs:
   include:
-    - name: "Linux build"
+    - name: "Linux build - x86"
```

Comment:
I'm thinking we should name these x86_64 to avoid potential confusion.

Reply:
Will do 👍